article/_posts/2022-06-16-What_Matters_Robot_Manipulation.md

```diff
@@ -0,0 +1,163 @@
+-  Multi-Human (MH). Transitions obtained from a group of six human operators with varying skill levels.
+
+Each dataset has both low-dimension observations and images from cameras.
+
+## Algorithms
+
+The authors considered six algorithms for the tasks:
+
+- BC\*: Behavior cloning (L2 loss on the action or KL divergence between the behavior policy and the learning policy (noted as Gaussian Mixture Models, GMM))
+- BC-RNN\*: Behavior cloning but with a recurrent policy
+- HBC: Hierarchical Behavioral Cloning, which includes two policies, a high-level one (with predicts future states) and low-level one (with predicts actions).
+- BCQ: Batch-Constrained Q-Learning, an offline RL algorithm.
+- CQL: Conservative Q-Learning, an offline RL algorithm.
+- IRIS: Implicit Reinforcement without Interaction, an offline RL algorithm. Also hierarchical, IRIS uses a RNN for its low-level policy.
+
+\* not actually reinforcement learning.
+
+## Experiments
+
+The authors devise 7 experiments to test the challenges proposed above:
+
+1. Algorithm Comparison (C1, C2): the authors test all the algorithms on all the datasets and tasks using low-dimension representations as states.
+2. Learning from Suboptimal Human Data (C2): the authors split the MH dataset into varying quality sub-datasets and train on them to see the effect of the trajectory quality
+3. Effect of Observation Space (C5): The authors evaluate the discrepancies in performance between learning from images or low-level features
+4. Selecting a Policy to Evaluate (C4): the authors evaluate the drops in performance when either selecting agents with the lowest validation loss or at the last checkpoint, instead of selecting them via online evaluation.
+5. Effect of Hyperparameter Choice (C5): The authors vary hyperparameters for the BC-RNN agent
+6. Effect of Dataset Size (C3): the authors vary the percentage of data used
+7. Applicability to Real-World Settings: the authors train on the real version of three tasks using hyperparameters found for the simulated versions.
+
+# Results
+
+## Exp. 1
+
+![](/article/images/what_matters_robotic/table1.jpeg)
+
+1.1 BC-RNN performs much better than BC
+
+1.2 Offline RL does not work on human data
+
+## Exp. 2
+
+![](/article/images/what_matters_robotic/table2.jpeg)
+
+Can-Paired is a balanced dataset where a single human operator purposely succeeded and failed 100 times each at the task.
+
+2.1 BC-RNN outperforms other algorithms except IRIS and HBC, which both also have some notion of temporality.
+
+2.2 CQL failed even in the Can-Paired test, where it should have been the best at distinguishing good from bad transitions.
+
+## Exp. 3
+
+![](/article/images/what_matters_robotic/fig2.jpeg)
+
+3.1 Agents trained on images can match agents trained on low-level features
+
+3.2 Adding extra features (like velocities) hurt the performance of agents
+
+3.3 Data augmentation on image inputs and other observations greatly improved results
+
+## Exp. 5
```

Comment:
Should this be exp 4 ?

Reply:
I put it there because the figure for exp 3 and 5 are joined, and exp4 has its own figure